examples/cra-kitchen-sink/config-addon-jest.json

```diff
@@ -5,6 +5,7 @@
   "transform": {
     ".*": "babel-jest"
   },
+  "transformIgnorePatterns": ["/node_modules/(?!lodash-es/.*)"],
```

Comment:
This one is not needed since we reverted the use of `lodash-es`

Reply:
you are right, I add it noted somewhere, forgot it ! nice catch thanks !